images/win/scripts/Installers/Install-Git.ps1

```diff
@@ -4,8 +4,12 @@
 ################################################################################
 
 # Install git
-# Temporary hardcode version 2.35.1 due to the issue with the actions\checkout https://github.com/actions/checkout/issues/760
-Choco-Install -PackageName git -ArgumentList '--version', "2.35.1.2", '--installargs="/VERYSILENT /NORESTART /NOCANCEL /SP- /CLOSEAPPLICATIONS /RESTARTAPPLICATIONS /o:PathOption=CmdTools /o:BashTerminalOption=ConHost /o:EnableSymlinks=Enabled /COMPONENTS=gitlfs"'
+Choco-Install -PackageName git -ArgumentList '--installargs="/VERYSILENT /NORESTART /NOCANCEL /SP- /CLOSEAPPLICATIONS /RESTARTAPPLICATIONS /o:PathOption=CmdTools /o:BashTerminalOption=ConHost /o:EnableSymlinks=Enabled /COMPONENTS=gitlfs"'
+
+Add-Content -Path "C:\Program Files\Git\etc\gitconfig" -Value "
```

Comment:
Why do not we use the same approach like we do for macOS?

`git config --system --add safe.directory "*"`

Reply:
@al-cheb on macOS the user remains the same after the image generation but on windows the setting will be erased as the user is different

Reply:
@miketimofeev , agree, what's why I put `system` flag.